pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -2126,7 +2126,7 @@ func TestChangefeedSchemaChangeBackfillCheckpoint(t *testing.T) {
 
 	cdcTestWithSystem(t, testFn, feedTestEnterpriseSinks)
 
-	log.Flush()
+	log.FlushFiles()
```

Comment:
nit: it looks like we do this a lot in tests.
```
log.FlushFiles()
log.FetchEntriesFromFiles(...)
```
It would be nice if you didn't have to worry about flushing in tests. Maybe this could be one function call to a test helper `FlushAndFetchEntriesFromFiles`. I don't feel too strongly about this though.